languages/en-US/_Base.json

```diff
@@ -1526,7 +1526,8 @@
     "LBL_CONDITION_WIZARD": "Condition wizard",
     "LBL_CURRENCY_CONVERTER": "Currency converter",
     "LBL_OK": "Ok",
-    "LBL_BATCH_ADDING_RECORDS": "Batch adding records"
+    "LBL_BATCH_ADDING_RECORDS": "Batch adding records",
+    "LBL_ATTACH_DOCUMENT_DESC": "When you select this option, it will attach PDF to the records. It doesn't work with Send email action"
```

Comment:
It doesn't work with Send email action.. -> not include.., or something like that